pysteps/io/exporters.py

```diff
@@ -460,8 +461,13 @@ def initialize_forecast_exporter_netcdf(
             + "'timestep' or 'member'"
         )
 
+    if isinstance(n_timesteps, list):
```

Comment:
this condition is checked multiple times later in the code, consider using a variable `n_timesteps_is_list = isinstance(n_timesteps, list)`